app/shared/src/buttons/__tests__/TransformButtonText.test.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import { Platform } from 'react-native';
+
+import TransformButtonText from '../TransformButtonText';
+
+describe('TransformButtonText', () => {
+  it('changes text to upper case on android', () => {
+    Platform.OS = 'android';
```

Comment:
You are leaking changed `Platform.OS` value to other tests. This is potentially dangerous...